src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,16 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userservice = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User saveduser = userservice.findByEmail(email);
+        if ((saveduser != null) && (email.equals(saveduser.getEmail()))
+                && (password.equals(saveduser.getPassword()))) {
+            return true;
+        } else {
+            return false;
+        }
+
```

Comment:
we do not need so many braces, and now, we can optimize it even more by removing the `if` and `else` completely and returning the result of this condition right away since it returns exactly what we need to return in our method, like so:
```suggestion
        return saveduser != null && email.equals(saveduser.getEmail())
            && password.equals(saveduser.getPassword());
```